README.md

```diff
@@ -15,6 +15,9 @@ yarn add rust-plugin
 | `debug` | Enable debug mode |
 | `wasmopt.level` | configure `wasm-opt` optimization level (default `z`) |
 | `wasmopt.required` | throw if the binary is not available (default `false`: warn) |
+| `wasmopt.snipRustPanickingCode` | Snip Rust's `std::panicking` and `core::panicking` code.(default `true`) |
+| `wasmopt.snipRustFmtCode` | Snip Rust's `std::fmt` and `core::fmt` code (default `true`). |
+| `wasmopt.functions` | Snip any function that matches the given regular expression or a string (default empty). |
```

Comment:
These should be `wasmsnip.*`, right?

Reply:
yes, good catch

Reply:
Oh, also: I don't think we should make snipping the panicking and fmt code on by default. It is pretty disastrous if you didn't intentionally do that, and you'll see your code just start trapping and throwing exceptions that are hard to debug :-p

Reply:
You're seeing an outdated diff, see a373939

Reply:
Great :)